drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/s3_zip_files_importer.rb

```diff
@@ -0,0 +1,71 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# This class iterates over the zip file objects in a bucket "directory" and
+# calls an importer on that directory after extraction.
+# FIXME: Does this already exist somewhere?
+
+module HmisExternalApis::AcHmis::Importers
+  class S3ZipFilesImporter
+    # FIXME: add some better exception handling for failure conditions
+    include NotifierConfig
+
+    attr_accessor :bucket_name
+    attr_accessor :prefix
+    attr_accessor :importer_class
+    attr_accessor :skip_lambda
+
+    def initialize(bucket_name: ENV.fetch('ACTIVE_STORAGE_BUCKET'))
```

Comment:
I wonder if we should require the bucket in production and only set `ENV.fetch('ACTIVE_STORAGE_BUCKET')` in development.  It makes me slightly nervous that we might be looking through the active storage bucket in production if we forget to pass this in.  That could have thousands of files in it.